plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoPush.test.ts

```diff
@@ -77,6 +97,10 @@ describe('github:repo:push', () => {
 
   beforeEach(() => {
     jest.resetAllMocks();
+
+    // @ts-ignore TS2339 Jest Mock is not detected by Typescript
+    initRepoAndPush.mockResolvedValue({ commitHash: 'test123' });
```

Comment:
Let's keep TS in the loop, or you might end up missing other type issues. For working around this particular case, see for example this: https://github.com/backstage/backstage/blob/d2ef67782534e7c02b367b383e5df242b89be026/plugins/catalog-backend-module-msgraph/src/processors/MicrosoftGraphOrgEntityProvider.test.ts#L47